cloudformation/eks/aws-eks-nodegroup_updateconfig.go

```diff
@@ -0,0 +1,18 @@
+package eks
+
+import "github.com/weaveworks/goformation/v4/cloudformation/types"
+
+// Nodegroup_UpdateConfig AWS CloudFormation Resource (AWS::EKS::Nodegroup.UpdateConfig)
+// TODO @nikimanoledaki: "See" with link to AWS CF docs
+type Nodegroup_UpdateConfig struct {
+
+	// MaxUnavailable AWS CloudFormation Property
+	// Required: false
+	// TODO: add link to docs
```

Comment:
TODO :smile: but I guess we will have to do this after its released?

Reply:
yeah, I've been postponing merging this because I've been hoping for this to appear in the docs, but they haven't yet. I'll keep an eye out next week once they officially release UpdateConfig so that I can add the docs then :)